lib/matplotlib/axes/_base.py

```diff
@@ -4601,3 +4614,28 @@ def get_shared_x_axes(self):
     def get_shared_y_axes(self):
         """Return a reference to the shared axes Grouper object for y axes."""
         return self._shared_axes["y"]
+
+    def set_capture_navigation(self, capture):
+        """
+        Set how pan/zoom events are forwarded to other axes.
+
+        Parameters
+        ----------
+        capture : bool or None
+            Possible values:
+
+            - True: Events are only executed on this axes.
+            - False: Forward events to other axes with lower or equal zorder
+            - None: Default behaviour ("True" for axes with an invisible patch
```

Comment:
`set_capture_navigation_events(None)` is quite opaque. Could we have a more telling name than None? There's precedence for a bool+str tristate e.g. in https://matplotlib.org/devdocs/api/_as_gen/matplotlib.axes.Axes.set_axisbelow.html?highlight=set_axisbelow#matplotlib.axes.Axes.set_axisbelow.

Something like this is understandable:
`set_capture_navigation_events('if-axes-is-non-transparent')`

Maybe you can find a shorter or better text?

Reply:
Hey @timhoffm, I can implement this but I'd very much like an agreement first to avoid having to change it multiple times...
Here are some suggestions from my side:
- `"axes_bg_visible"`, `"if_axes_bg_visible"`
- `"visible_background"`, `"visible_bg"`, `"background_visible"` or `"bg_visible"` 
  (it's a method of an axes... so it should be clear this refers to the axes)

I'm happy with either one (even your suggestion... however I find it a bit long and it's not about the visibility of the axes but about the visibility of the background patch of the axes).

Reply:
hey @timhoffm ... just checking in to make sure it keeps rolling 😅

Reply:
Sorry, this got lost in the stream of notifications. Thanks for nudging.

I agree that "axes" can be implied. I suggest `capture_navigation_events='if_background_visible'`.

Yes, it's a bit long, but it's quite self-explanatory now. And it's rare that users have to write it themselves because it's the default.

Reply:
Is `'auto'` sufficient?

Reply:
`"auto"` is now implemented ...